package.json

```diff
@@ -54,7 +54,8 @@
     "normalize-path": "^3.0.0",
     "p-retry": "^4.6.1",
     "spdx-license-ids": "^3.0.10",
-    "stream-chunker": "^1.2.8"
+    "stream-chunker": "^1.2.8",
+    "temp-dir": "^2.0.0"
```

Comment:
We shouldn't be creating a directory, just a one-file that get's updated on each deploy with the latest manifest. So temp-dir isn't required.